From: Tzafrir Cohen <tzafrir@debian.org>
Subject: Use Python 3
Forwarded: no

This change is done (conditionally) in Upstream's spec in the %prep stage.

--- a/py/mock.py
+++ b/py/mock.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python -tt
+#!/usr/bin/python3 -tt
 # -*- coding: utf-8 -*-
 # vim:expandtab:autoindent:tabstop=4:shiftwidth=4:filetype=python:textwidth=0:
 
--- a/py/mockchain.py
+++ b/py/mockchain.py
@@ -1,4 +1,4 @@
-#!/usr/bin/python -tt
+#!/usr/bin/python3 -tt
 # -*- coding: utf-8 -*-
 # vim: noai:ts=4:sw=4:expandtab
 
